bindings/python/iota_sdk/types/output.py

```diff
@@ -107,8 +107,8 @@ class AccountOutput:
     account_id: HexStr
     state_index: int
     foundry_counter: int
-    unlock_conditions: List[Union[StateControllerAddressUnlockCondition,
-                                  GovernorAddressUnlockCondition]] = field(
+    unlock_conditions: List[Union[AddressUnlockCondition,
```

Comment:
lint is going to complain about the solo union

Reply:
What should it be then ?

Reply:
Just the single type

Reply:
a list

Reply:
Ok I can do that, but wouldn't it be better to add an ignore? Sounds a bit error prone without the union, we'd definitely forget to update, what do you think?

Reply:
I don't think it matters honestly. Python wouldn't tell us anyway.

Reply:
I thought we had a Union of 1 somewhere, but cant find it. Lets change to the list of 1 type then

Reply:
Done